WORKSPACE

```diff
@@ -67,12 +64,31 @@ bind(
     actual = "//tools:java_toolchain",
 )
 
+http_archive(
+    name = "rules_java",
```

Comment:
What is this needed for? Can you add a comment before it to explain if it's needed for a later dependency.

Reply:
This was an error `Failed to load Starlark extension '@rules_java//java:repositories.bzl'.`from a missing definition `rules_java`. But I don't actually think we need rules_java, when I remove [these](https://github.com/oppia/oppia-android/blob/afa115c6bf0b74c58c71dbf5be6cc02c94f892fa/WORKSPACE#L67-L77) lines it still builds.

Do you know what `rules_java_dependencies()` and `rules_java_toolchains()` is doing? Maybe this used to import the `javalite_toolchains` for us?

Reply:
I'm assuming this is for the Java Starlark rules (e.g. java_library). I just wanted to make sure we contextualize why it's needed so others don't wonder in the future.